src/components/SQForm/useFormButton.tsx

```diff
@@ -12,11 +12,12 @@ type UseFormButtonReturnType = {
   isValid: boolean;
   handleClick: DebouncedState<React.MouseEventHandler<HTMLButtonElement>>;
   [key: string]: unknown;
+  handleReset: () => void;
 };
 
 export function useFormButton(
   isDisabled: boolean,
-  shouldRequireFieldUpdates: boolean,
+  shouldRequireFieldUpdates?: boolean,
```

Comment:
I noticed this component never had any propTypes to start with that would tell me if these props should be required or optional, i just looked inside of some other components that are consuming this to determine that, some use all 3 props and others. just seem to use IsDiabled, so i went ahead and made this 'shouldRequireFieldUpdates' optional.